server/admin_management/api/app/routers/router_application_admin.py

```diff
@@ -58,6 +59,7 @@ def create_application_admin(
     request: Request,
     token_claims: dict = Depends(jwt_validation.authorize),
     requester: Requester = Depends(get_current_requester),
+    target_user: TargetUser = Depends(get_verified_target_user),  # validate target user
```

Comment:
so we could verify the target user here. But I'm seeing we also try to get the target user from database for audit. But I feel for audit purpose, it's okay to use the target_user here directly? It will just tell from the given parameter, we're trying to grant access to which target user. In the current situation, if we want to grant access to a new user, it won't return from the user_service call, the user does not exist yet. But I feel it's okay to use the target_user here directly, for both grant app admin and grant delegated admin

<img width="831" alt="Screen Shot 2024-05-27 at 12 37 19 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/7ca39ef4-58f0-4d28-8fdc-1a459ecf4743">